addons/hr_timesheet/report/project_report.py

```diff
@@ -17,7 +17,7 @@ def _select(self):
         select_to_append = """,
                 (t.effective_hours * 100) / NULLIF(t.planned_hours, 0) as progress,
                 t.effective_hours as hours_effective,
-                t.planned_hours - t.effective_hours - t.subtask_effective_hours as remaining_hours,
+                NULLIF(t.remaining_hours, 0) as remaining_hours,
```

Comment:
Since Alexandra noted that the computation is still wrong. I suggest looking further into it and why there is a difference between what is in the db and what we show in the views.

Reply:
I have already checked in db, issue is not in db issue is in count, project.task pivot view and project report task count is not same that's why we get wrong calculation.

Reply:
Okay, the difference between the value reported should be addressed nonetheless. I don't think it matters whether the problem is in the value calculation or the count.